src/main/java/com/terraformersmc/cinderscapes/init/CinderscapesPlacementModifierTypes.java

```diff
@@ -10,11 +10,9 @@
 
 // TODO: Check
 public class CinderscapesPlacementModifierTypes {
-    public static PlacementModifierType<SafelistRangeCeilingPlacementModifier> COUNT_CEILING;
-    public static PlacementModifierType<SafelistRangeFloorPlacementModifier> COUNT_FLOOR;
+    public static final PlacementModifierType<SafelistRangeCeilingPlacementModifier> COUNT_CEILING = Registry.register(Registry.PLACEMENT_MODIFIER_TYPE, new Identifier("cinderscapes", "safelist_range_ceiling"), () -> SafelistRangeCeilingPlacementModifier.UNIFORM_CODEC);
+    public static final PlacementModifierType<SafelistRangeFloorPlacementModifier> COUNT_FLOOR  = Registry.register(Registry.PLACEMENT_MODIFIER_TYPE, new Identifier("cinderscapes", "safelist_range_floor"), () -> SafelistRangeFloorPlacementModifier.UNIFORM_CODEC);
```

Comment:
The registry can still be modified during initialization rather than class loading.